src/ReactiveDomain/ReactiveDomain.Tests/Logging/when_logging_high_volume_message_traffic.cs

```diff
@@ -0,0 +1,154 @@
+        private int _countedEventCount;
+        private int _testDomainEventCount;
+        private int _numberOfItemsLogged;
+        private int _catchupSubscriptionMsgs;
+
+        private TestCommandSubscriber _cmdHandler;
+        protected override void When()
+        {
+            // commands must have a commandHandler
+            _cmdHandler = new TestCommandSubscriber(Bus);
+
+             _commandFireCount = 0;
+             _commandAckCount = 0;
+             _commandSuccessCount = 0;
+             _lastCommandCount = 0;
+             _countedEventCount = 0;
+             _testDomainEventCount = 0;
+             _numberOfItemsLogged = 0;
+             _catchupSubscriptionMsgs = 0;
+
+            _listener = Repo.GetListener(Logging.FullStreamName);
+            _listener.EventStream.Subscribe<Message>(this);
+
+            _listener.Start(Logging.FullStreamName);
+
+            // create and fire a mixed set of commands and events
+            for (int i = 0; i < _maxCountedMessages; i++)
+            {
+                Bus.Publish(
+                    new CountedEvent(i,
+                        Guid.NewGuid(),
+                        Guid.NewGuid()));
+
+                // this is just an example command - choice to fire this one was random
+                var cmd = new InformUserCmd("title",
+                                        $"message{i}",
+                                        Guid.NewGuid(),
+                                        null);
+                Bus.Fire(cmd,
+                    $"exception message{i}",
+                    TimeSpan.FromSeconds(2));
+
+                Bus.Publish(new TestDomainEvent(Guid.NewGuid(), Guid.NewGuid()));
+
+            }
+
+            var tstCmd = new TestCommands.TestCommand3(
+                        _correlationId,
+                        null);
+
+            Bus.Fire(tstCmd,
+                "TestCommand3 failed",
+                TimeSpan.FromSeconds(2));
+
+        }
+
+
+        [Fact(Skip = "pending deletion of log stream")]
+        public void can_verify_high_volume_traffic_logged()
+        {
+            // Wait for last command to be queued
+            TestQueue.WaitFor<TestCommands.TestCommand3>(TimeSpan.FromSeconds(10));
+
+            // Wait  for last command to be "heard" from logger/repo
+            Assert.IsOrBecomesTrue(() => 
+                _lastCommandCount == 1, 
+                40000,
+                $"Last command count = {_lastCommandCount}. Command never handled");
+
+            // Wait  for last CountedEvent to be "heard" from logger/repo
+            Assert.IsOrBecomesTrue(
+                () => _countedEventCount == _maxCountedMessages, 
+                2000,
+                $"Message {_countedEventCount} doesn't attain expected index {_maxCountedMessages}");
+
+            Assert.True(_countedEventCount == _maxCountedMessages, $"Message {_countedEventCount} doesn't match expected index {_maxCountedMessages}");
```

Comment:
Use Assert.Equal instead. I'm sure there are other places in this pull request where the same thing applies, but I'm not going to comment on all of them.

Reply:
we can add an override to assert equal in reactive domain testing to add a custom message.

Reply:
A fine idea for another sprint/requirement.